script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -502,18 +502,19 @@ She charges Vlov head-on, despite knowing that his strange weapon has the potent
 [sha:fb7cb99ac72313773eb97ec08ff3a77f8645767e]{
 -- Page 551, Offset 10079.
 -- 「だが貴様は知るまい。
-"But you do not understand.
+"Yet you do not know...
 }
 [sha:5a34da987dbe2ad8e37cd710badd8724996c6c5a]{
 -- Page 551, Offset 10080.
 -- <彼|か>の領土はこの星の一部でありながら、“生きるに<能|あた>わず”と除外された生命圏。
 // I think this か is an inanimate reference here?
-Although that territory is a part of this planet, it is an isolated biosphere in which life cannot exist.
+// just a reading, dont worry, I got it. (P)
+Although this a part of the planet be, yet still a biosphere foreclosed... in which none may live...
 }
 [sha:d5e217d9d0ade899bb9bf162e4b8b308f788e254]{
 -- Page 551, Offset 10081.
 -- いかに真祖の姫であろうと、この地獄に耐えきれるか……！」
-Even a Princess of the True Ancestors like yourself cannot possibly endure a hell like this...!"
+Though Princess of the True Ancestors you may be... think not this hell you can endure...!
```

Comment:
Yeah some of these Vlov lines read _really_ extra, unless the JP really justifies it in a way I can't see I might dial it back a bit